templates/web/base/admin/emergencymessage/edit.html

```diff
@@ -47,6 +47,7 @@
 
   [% IF hardcoded_reporting_message %]
     <p>Your reporting page message is currently hard-coded.</p>
+    <pre>[% hardcoded_reporting_message %]</pre>
```

Comment:
I was originally thinking of a rendered preview, but actually showing the HTML makes a lot more sense!